lib/lrama/grammar/parameterizing_rule_rhs_builder.rb

```diff
@@ -8,6 +8,46 @@ def initialize
         @user_code = nil
         @precedence_sym = nil
       end
+
+      def build_rules(token, actual_args, parameters, rule_counter, lhs, lhs_tag, line, rule_builders)
+        nested_rules = build_nested_rules(token, actual_args, parameters, rule_counter, lhs_tag, line, rule_builders)
+        rule = Rule.new(id: rule_counter.increment, _lhs: lhs, _rhs: rhs(token, actual_args, parameters, nested_rules.last), lhs_tag: lhs_tag, token_code: user_code, precedence_sym: precedence_sym, lineno: line)
+        ParameterizingRule.new(rules: nested_rules.map(&:rules) + [rule], token: lhs)
+      end
+
+      private
+
+      def build_nested_rules(token, actual_args, parameters, rule_counter, lhs_tag, line, rule_builders)
```

Comment:
[note] In the future, `ParameterizingRuleRhsBuilder` might delegate rule building to `RuleBuilder` so that RHS replacement logic is put into only one place.